censys/asm/assets/assets.py

```diff
@@ -29,6 +29,7 @@ def get_assets(
         tag: Optional[List[str]] = None,
         tag_operator: Optional[str] = None,
         source: Optional[List[str]] = None,
+        discovery_trail: Optional[bool] = None
```

Comment:
I think black wants a comma at the end of this line.

```suggestion
        discovery_trail: Optional[bool] = None,
```

Reply:
Just updated pull request with that change.